doc/release/README.prereqs

```diff
@@ -4,7 +4,7 @@ Chapel Prerequisites and Assumptions
 
 If you have a standard UNIX environment, a C/C++ compiler, some basic
 scripting languages, a GNU-compatible make, and awk installed you should
```

Comment:
Hi @noakesmichael : Did I misunderstand our lunchtime conversation?  It looks like 'awk' is still named in this line and  the three at the bottom of the file?  Maybe there's another pending pull request I haven't seen yet...?